packages/devtools_app/test/inspector_layout_tab_test.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:devtools_app/src/inspector/flutter/layout_tab.dart';
+import 'package:devtools_app/src/ui/fake_flutter/_real_flutter.dart';
```

Comment:
use
'package:flutter/material.dart'
here and elsewhere.
'_real_flutter' and 'fake_flutter.dart' were just for porting the devtools code.